docs/fdc3-compliance.md

```diff
@@ -26,12 +26,13 @@ An application provider is largely a downstream consumer of FDC3 standards. It M
 Depending on persona, implementation compliance with FDC3 will mean different things.
 
 ### Platform Provider
-To meet the requirements of FDC3, a platform provider would need to support the following:
+For platform providers FDC3 compliance requires:
 
-* Connection to 1 or more App Directories meeting the FDC3 App Directory standards
-* Provide an API to all applications running in the context of the platform that meets the FDC3 API standards, including:
+* Support for connections to 1 or more App Directories meeting the FDC3 App Directory standards SHOULD be provided
```

Comment:
This section was missing imperatives (bar one on workflows). Matched style with Application provider section as one was needed in the `window.fdc3` addition